sound/soc/sof/ipc4-topology.c

```diff
@@ -1801,6 +1801,7 @@ static int sof_ipc4_dai_config(struct snd_sof_dev *sdev, struct snd_sof_widget *
 	case SOF_DAI_INTEL_HDA:
 		gtw_attr = ipc4_copier->gtw_attr;
 		gtw_attr->lp_buffer_alloc = pipeline->lp_mode;
+		pipeline->skip_fe_trigger = true;
```

Comment:
skip_during_fe_trigger?

I just don't see how a BE can skip an FE, all DPCM actions are initiated by FEs.

Also what is so special about HDaudio that requires a change here?